ably/README.md

```diff
@@ -0,0 +1,73 @@
+## Overview
+The [Ably][1] platform is used to power real-time use cases such as multiplayer, chat, data synchronization, data broadcast, and notifications for highly scalable web and mobile applications around the world. Using our APIs, engineers are free to focus on building core functionality, rather than having to provision and maintain servers and cloud infrastructure.
+
+The Ably Datadog Integration sends [Ably statistics][2] metrics directly to your Datadog account.
+
+Using Ably's Datadog Integration, you can:
+- Use [Ably statistics][2] alongside other key metrics in Datadog
+- Correlate Ably message, channel, and connection usage for collaborative analysis in Dashboards
+- Identify unexpected activity and troubleshoot potential issues with your applications powered by Ably
```

Comment:
Can you clarify what the last line means? "Identify unexpected activity and troubleshoot potential issues with your applications powered by Ably"

Reply:
I've amended this line to "View and track Ably usage statistics in Datadog" in 0d904f84. Hopefully that's clearer.